dydx/dydxFormatter/dydxFormatterTests/dydxFormatterTests.swift

```diff
@@ -18,6 +18,32 @@ final class dydxFormatterTests: XCTestCase {
         // Put teardown code here. This method is called after the invocation of each test method in the class.
     }
 
+    
+    func testDecimalRaw() {
+        struct TestCase {
+            let number: NSNumber
+            let digits: Int
+            let expected: String
+            var locale: Locale = Locale.current
+        }
+
+        let testCases: [TestCase] = [
+            .init(number: 1, digits: 2, expected: "1.00"),
+            .init(number: -0.001, digits: 0, expected: "0"),
+            .init(number: -0.001, digits: 3, expected: "-0.001"),
+            .init(number: -0.001, digits: 2, expected: "0.00"),
+            .init(number: 0.001, digits: 2, expected: "0.00"),
+            .init(number: -0.005, digits: 2, expected: "-0.01"),
+            .init(number: -0.0051, digits: 2, expected: "-0.01"),
+            .init(number: 1123345.123, digits: 2, expected: "1123345.12", locale: Locale(identifier: "fr_FR")),
+        ]
+
+        for testCase in testCases {
+            let formatted = dydxFormatter.shared.decimalRaw(number: testCase.number, digits: testCase.digits)
```

Comment:
@ruixhuang locale does not seem to be used here (from last test case)

Reply:
Calling the wrong function.. updating...